pkg/controller/dbsyncer/subscription_db_to_transport_syncer.go

```diff
@@ -0,0 +1,38 @@
+package dbsyncer
+
+import (
+	"fmt"
+	"time"
+
+	appsv1 "github.com/open-cluster-management/multicloud-operators-subscription/pkg/apis/apps/v1"
+	"github.com/stolostron/hub-of-hubs-spec-transport-bridge/pkg/bundle"
+	"github.com/stolostron/hub-of-hubs-spec-transport-bridge/pkg/db"
+	"github.com/stolostron/hub-of-hubs-spec-transport-bridge/pkg/intervalpolicy"
+	"github.com/stolostron/hub-of-hubs-spec-transport-bridge/pkg/transport"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+const (
+	subscriptionsTableName     = "subscriptions"
+	subscriptionRuleMessageKey = "subscriptions"
+)
+
+// AddSubscriptionsDBToTransportSyncer adds subscriptions db to transport syncer to the manager.
+func AddSubscriptionsDBToTransportSyncer(mgr ctrl.Manager, db db.SpecDB, transport transport.Transport,
+	syncInterval time.Duration) error {
+	if err := mgr.Add(&genericDBToTransportSyncer{
+		log:                ctrl.Log.WithName("subscription-db-to-transport-syncer"),
+		db:                 db,
+		dbTableName:        subscriptionsTableName,
+		transport:          transport,
+		transportBundleKey: subscriptionRuleMessageKey,
+		intervalPolicy:     intervalpolicy.NewExponentialBackoffPolicy(syncInterval),
+		createObjFunc:      func() metav1.Object { return &appsv1.Subscription{} },
+		createBundleFunc:   bundle.NewBaseBundle,
```

Comment:
pay attention to the order of fields when you initialize the struct.
should be the same order they appear in the struct (and the same as in all other db syncers).
again, mainly for consistency. keeping consistent helps navigating in code more easily.
see the order for example here:
https://github.com/stolostron/hub-of-hubs-spec-transport-bridge/blob/main/pkg/controller/dbsyncer/placementbindings_db_to_transport_syncer.go#L22-L32